src/components/Authentication/ForgotPassword.jsx

```diff
@@ -1,42 +1,74 @@
+import { Box, Container, Center, Text, Flex, Heading, Input, Button } from '@chakra-ui/react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Center' is defined but never used.